Xamarin.Forms.Core/HandlerAttribute.cs

```diff
@@ -5,19 +5,60 @@ namespace Xamarin.Forms
 	[AttributeUsage(AttributeTargets.Assembly, AllowMultiple = true)]
 	public abstract class HandlerAttribute : Attribute
 	{
+		protected HandlerAttribute(Type handler, Type target, object[] filteredSdkVersionInformation) : this(handler, target)
```

Comment:
we don't do 'magic' `object[]` parameter in public API. it is likely to throw ~~on line 33~~ on `Validate()`, if the array doesn't have the right size, or the worn element types

Also, there's no need to add ctor parameters to the attribute. you can add properties, and those will be available as optional named param when you declare the attribute. That's the beauty, and weirdness, of attributes: attributes had optional parameters long before C# got it, and `DebuggerDisplayAttribute` had string interpolation long before c#6...